lib/src/roulette_unit.dart

```diff
@@ -21,6 +21,7 @@ class RouletteUnit {
   const RouletteUnit({
```

Comment:
If we use this constructor to specify both icon and text, the text would be overiden by the icon, isn't it? Add some `assert` here would be better.

Reply:
Yes, you cannot have both text and icon since the text is not painted when an icon is provided

Reply:
Added `assert`